appveyor.yml

```diff
@@ -42,4 +42,5 @@ build: false
 test_script:
   - cargo test
   - cargo test --no-default-features
-  - cargo test --no-default-features --features "deflate-miniz"
+  - cargo test --no-default-features --features "deflate"
+  - cargo test --no-default-features --features "bzip2"
```

Comment:
@mvdnes you made comments on the travis build. Should appveyor build each feature individually?

Reply:
I've removed the extra builds in appveyor.